### DISCUSSION TITLE
eps32 parent request:no buffs error

### DISCUSSION THREAD

**Discussion by mertcivan** [community/community#32753] [Other Feature Feedback, Questions, & Ideas]
Hi, I am using esp32 as border router and I can connect only 10 end devices to thread network.When I tried to connect  eleventh device I am getting parent request :no buffs if you answer , I would be happy. Thank you in adnvance

**Comment by RedoxOwn** [comment]
That's because the default maximum number of sockets that can be created is 10
According to **espressif** documentation, esp32 can support up to 32 sockets
I haven't tested it, but it seems one way to change the default maximum sockets is by modifying the file **sdkconfig.h** located in **app/build/include/sdkconfig.h**
#define CONFIG_LWIP_MAX_SOCKETS 10
Change the default value **10** to whatever you want and recompile
Example:
from
#define CONFIG_LWIP_MAX_SOCKETS 10
to
#define CONFIG_LWIP_MAX_SOCKETS 11

Related links:
[https://espressif-docs.readthedocs-hosted.com/projects/espressif-esp-faq/en/latest/software-framework/wifi.html#lwip-what-is-the-maximum-number-of-sockets-that-can-be-created](https://espressif-docs.readthedocs-hosted.com/projects/espressif-esp-faq/en/latest/software-framework/wifi.html#lwip-what-is-the-maximum-number-of-sockets-that-can-be-created)
[https://myesp8266.blogspot.com/2016/11/esp32-increase-maximum-number-of-sockets.html](https://myesp8266.blogspot.com/2016/11/esp32-increase-maximum-number-of-sockets.html)
